modules/recommended-configurable-storage-technology.adoc

```diff
@@ -121,8 +121,19 @@ It is not recommended to use file storage for a hosted metrics cluster deploymen
 
 In an {product-title} hosted logging cluster deployment:
 
-* The preferred storage technology is block storage.
-* Object storage is not configurable.
+* Loki Operator:
+** The preferred storage technology is S3 compatible Object storage.
+** Block storage is not configurable.
+
+* Elasticsearch Operator:
+** The preferred storage technology is block storage.
+** Object storage is not configurable.
+
+[NOTE]
+====
+As of logging version 5.4.3 the OpenShift Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator.
```

Comment:
```suggestion
As of logging version 5.4.3 the OpenShift Elasticsearch Operator is deprecated and will be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator.
```

Reply:
Suggestion based on the first sentence which states it (Openshift Elasticsearch Operator) will be removed, so not sure if that needs to be stated again.

Reply:
This snippet is pre-existing, and is worded in accordance with the [supplementary style guide ](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes)requirements for deprecation notices.